src/contracts/interfaces/GPv2Authentication.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+interface GPv2Authentication {
+    function isSolver(address prospectiveSolver) external view returns (bool);
```

Comment:
MIssing doc-comments for interface and method. This is particularly important as this is the interface the GPv2Settlement contract expects.

Reply:
Yes! Very nice. Have included some, hopefully, appropriate doc strings.

Reply:
Looks great!